docker-compose.full.yml

```diff
@@ -79,42 +59,8 @@ services:
             - 8080:80
         volumes:
             - ./devTools/docker/vhosts.conf:/etc/nginx/conf.d/default.conf
-            - ./wordpress:/app
         depends_on:
             - fpm
```

Comment:
`web` references the `fpm` container and `vhosts.conf`, which both don't exist anymore. `web` is not needed at this point though, so we should just remove it.